test/README.md

```diff
@@ -21,7 +21,7 @@ os::test::junit::declare_suite_start "test/<TESTNAME>"
 
 ## Role of Companion scripts in hack/testing
 
-The wrapper or companion script for each test is responsible for 
+The wrapper or companion script for each test is responsible for
```

Comment:
@stevekuznetsov is this true?  I thought the purpose of the `hack/testing/test-*` scripts was to bridge us until we could get rid of `hack/testing`

Reply:
If the only setup, etc that is necessary lives in `entrypoint.sh` then yes, these are not necessary. I think some of these scripts right now call into the test 2x, one for ops and one for not

Reply:
That's kind of a question of how you want to structure your test cases TBH and I don't really have a strong opinion -- but it does seem cleaner to say "we expect these components to work this way" then call that test twice with two different sets of components

Reply:
> I think some of these scripts right now call into the test 2x, one for ops and one for not

I don't think so.  Some of the tests, like curator.sh, will run with and without ops _inside_ the test script itself.

Reply:
Yeah, looks like drift has happened -- whatever you feel is appropriate here should be enforced so it's a little more clear